### ISSUE TITLE
JulianDay is inconsistent across versions

### ISSUE THREAD

**Issue by zabetak** [igr/julian-day#3]
The date `2023-05-23` in previous releases it was mapped to the Julian day `2460088` while now (7.0.0) it is mapped to either `2460087`  or `2460100`.

#### org.jodd:jodd-core:3.5.2
```java
import jodd.datetime.JDateTime;
new JDateTime(2023,5,23).getJulianDayNumber() //2460088 
```
#### org.jodd:jodd-util:6.0.0
```java
import jodd.time.JulianDate;
JulianDate.of(LocalDate.of(2023,5,23)).getJulianDayNumber() //2460088 
```
#### org.jodd:julian-day:7.0.0 
```java
import jodd.julianday.JulianDateTime;
import jodd.julianday.JulianDay;
JulianDay.ofGregorianDate(LocalDate.of(2023,5,23)).day() //2460087
JulianDay.ofJulianDate(new JulianDateTime(2023,5,23,0,0,0,0)).day() //2460100 
```
I am not sure which number is the correct one but the fact they are different creates compatibility problems when data are persistent and reading/writing is performed cross versions.

**Comment by igr** [comment]
No worries, I know what it is. I will add a method shortly and release it!

The old `getJulianDayNumber()` is just the integer number of a day, but with a little twist (as Julian days start at noon). I will add a similar method to the new library.

You probably do not need Julian calendar dates. If you were not considering them explicitly, just use "gregorian" methods.

**Comment by igr** [comment]
The thing is that `day()` is not JDN. Added a method that calculates the jdn.

**Comment by igr** [comment]
Published 7.1.0
